llama_hub/earning_calls_transcripts/README.md

```diff
@@ -0,0 +1,86 @@
+# EARNING CALL TRANSCRIPTS LOADER
+
+This loader fetches the earning call transcripts of US based companies from the website [discountingcashflows.com](https://discountingcashflows.com/). It is not available for commercial purposes
+
+Install the required dependencies
+
+```
+pip install -r requirements.txt
+```
+
+The Earning call transcripts takes in three arguments
+
+* Year
+* Ticker symbol
+* Quarter name from the list ["Q1","Q2","Q3","Q4"]
+
+## Usage
+
+```python
+from llama_index import download_loader
+
+IMDBReviewsloader = download_loader('EarningCallTranscripts')
```

Comment:
nit: wrong name

Reply:
Hi @jerryjliu, thanks for your input
Yes I have done the nit changes and README ones too